steely/plugins/choose.py

```diff
@@ -0,0 +1,18 @@
+'''for those indecisive moments'''
+
+__author__ = 'itsdoddsy'
+COMMAND = 'choose'
+
+import re
+import random
+
+ERRORS = ['fucking moron',
+          'how about no',
+          'pikachu',]
+
+def main(bot, author_id, message, thread_id, thread_type, *args, **kwargs):
+    try:
+        args = re.findall(r"\w+\b(?<!\bchoose)", message)

```

Comment:
This regex could be compiled

Reply:
Fuck off

Reply:
you also forget i have next to no knowledge of python

Reply:
I don't think that's a big deal

Reply:
not a big deal, but a good olde nit pick